src/index.html

```diff
@@ -1,5 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+  <title>freeCodeCamp Manila</title>
+</head>
```

Comment:
You have duplicated `head` tag, see line 32

Reply:
I update the src/index.html and remove the duplicated head tag

Reply:
Hey @rcode321, I think you have removed the wrong head tag the current styles have been replaced.

Here's a screenshot
![localhost_1234-freeCodeCampManila](https://user-images.githubusercontent.com/25174423/66704026-22948680-ed4b-11e9-863e-8695560f21e0.png)